Makefile

```diff
@@ -246,6 +248,23 @@ bundle-build: bundle ## Build the bundle image.
 bundle-push: ## Push the bundle image.
 	$(MAKE) docker-push IMG=$(BUNDLE_IMG)
 
+.PHONY: operator-sdk
```

Comment:
I personally prefer to keep the dev prerequisites (golang, operator-sdk etc) outside of the Makefile, since there could be different ways to install and configure it. For example operator-sdk might be on my system path for usage across different operators.
Instead, checking for the presence of specific version of operator-sdk binary (based on osc operator requirement) in the Makefile and throwing error is portable and useful imho.

Reply:
Sounds reasonable.

Does this work?
1. check if operator-sdk already exists in the path
    - download into ./bin IFF it doesn't
 2.  check the version of operator-sdk against $OPERATOR-SDK-VERSION (allow overrides)
     - fail make if version isn't matched

Reply:
I'm leaving the operator-sdk section unchanged